static/src/javascripts/projects/common/modules/ui/cmp-ui.js

```diff
@@ -81,12 +45,11 @@ export const addPrivacySettingsLink = (): void => {
 export const consentManagementPlatformUi = {
```

Comment:
i'm not sure how much more work it would be, but i'd be really inclined to remove this. it feels like it only exists because it was needed before, rather than being needed now?

Reply:
discussed offline, one for a further PR

Reply:
I'll open the PR tomorrow: https://github.com/guardian/frontend/compare/cmp-remove-v3-cmp-cleanup